### ISSUE TITLE
4 races definition and training

### ISSUE THREAD

**Issue by lzhangbj** [dchen236/FairFace#19]
Thank you for your work. May I know how did you define the 4-races and how did you train it?

In the paper, it seems like you merged white and middle eastern into white, and merged east asian and southeast asian into asian, and then train the model on these 4 classes. However, the last fc layer of your pretrained model still predicts 18 classes (https://github.com/dchen236/FairFace/blob/74b4f93e527f1fb2b27b2b425227c3ded0521830/predict.py#L71) and you only use the first 4 values for race prediction (https://github.com/dchen236/FairFace/blob/74b4f93e527f1fb2b27b2b425227c3ded0521830/predict.py#L134). 
So how do you define these pretrained fc layer and how did your merge the races?

And moreover, how do you define the races groups in table 3&4 in the arxiv paper and how did you conduct the experiments? Thank you!

**Comment by yuhaoban** [comment]
Same confusion here. Based on the author's previous response in other issue tickets, I understand you can't release the datasets that are not under cc license but why not the training script and detailed parameters for training? Could you release this information for testing, please?

**Comment by akashAD98** [comment]
yes i also want to know how to train this network & what are the parameters you used ?
